docs/building_pipelines.rst

```diff
@@ -293,3 +294,423 @@ Warnings raised will also be logged to the error log but will not cause data to
+       {
+         "name": "Branch",
+         "readings": [
+           {
+             "asset_code": "latheCurrent",
+             "user_ts": "2025-04-01 08:18:42.262970+00:00",
+             "ts": "2025-04-01 08:18:42.262970+00:00",
+             "reading": {
+               "current": 750
+             }
+           },
+           {
+             "asset_code": "latheIR",
+             "user_ts": "2025-04-01 08:18:42.262975+00:00",
+             "ts": "2025-04-01 08:18:42.262975+00:00",
+             "reading": {
+               "gearbox": 27.9788,
+               "motor": 29.9785,
+               "headstock": 23.65,
+               "tailstock": 20.1666666667,
+               "tool": 18.73
+             }
+           },
+           {
+             "asset_code": "lathe",
+             "user_ts": "2025-04-01 08:18:42.262955+00:00",
+             "ts": "2025-04-01 08:18:42.262955+00:00",
+             "reading": {
+               "rpm": 349,
+               "x": 0,
+               "depth": 40,
+               "state": "Spining Up"
+             }
+           }
+         ]
+       },
+       [
+         {
+           "name": "CurrentOnly",
+           "readings": [
+             {
+               "asset_code": "latheCurrent",
+               "user_ts": "2025-04-01 08:18:42.262970+00:00",
+               "ts": "2025-04-01 08:18:42.262970+00:00",
+               "reading": {
+                 "current": 750
+               }
+             },
+             {
+               "asset_code": "latheIR",
+               "user_ts": "2025-04-01 08:18:42.262975+00:00",
+               "ts": "2025-04-01 08:18:42.262975+00:00",
+               "reading": {
+                 "gearbox": 27.9788,
+                 "motor": 29.9785,
+                 "headstock": 23.65,
+                 "tailstock": 20.1666666667,
+                 "tool": 18.73
+               }
+             },
+             {
+               "asset_code": "lathe",
+               "user_ts": "2025-04-01 08:18:42.262955+00:00",
+               "ts": "2025-04-01 08:18:42.262955+00:00",
+               "reading": {
+                 "rpm": 349,
+                 "x": 0,
+                 "depth": 40,
+                 "state": "Spining Up"
+               }
+             }
+           ]
+         },
+     ...
+
+The example has been truncated at the second node merely to save space in the documentation.
+
+suspendIngest
+#############
+
+When the debugger is attached to a service it does not stop the service from ingesting and processing new data, it merely adds a way to view the latest data as it traverse the pipeline. In situation where the user wishes to troubleshot the operation of the pipeline it may be desirable to suspend the service from ingesting new data. This allows the data to be examined, configuration to be modified and data resent through the pipeline to observe the impact of configuration changes. To stop new data ingesting into the pipeline use the *suspendIngest* command.
```

Comment:
- processing new data, it merely adds -> processing new data; it merely adds
- troubleshot -> troubleshoot